package.json

```diff
@@ -20,6 +20,6 @@
   },
   "homepage": "https://github.com/NielsSteensma/Dhalang#readme",
   "dependencies": {
-    "puppeteer": "^9.0.0"
+    "puppeteer": "^13.3.2"
```

Comment:
The previous version of Puppeteer doesn't accept the 'timeout' param during a 'printToPDF' call.